clients/drcachesim/tracer/tracer.cpp

```diff
@@ -1055,6 +1055,11 @@ event_app_instruction(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst
             FATAL("Fatal error: failed to reserve aflags\n");
     }
 
+    bool needs_instru = false;
+    if (instr_is_label(instr) && instr_get_note(instr) == (void *)DR_NOTE_RSEQ_ENTRY) {
```

Comment:
nit: extract into separate bool since it's reused below as well.

Reply:
Done.